code/datums/outfits/outfit_admin.dm

```diff
@@ -7,6 +7,13 @@
 		H.mind.assigned_role = name
 		H.job = name
 
+/datum/outfit/admin/post_equip(mob/living/carbon/human/H, visualsOnly)
+	. = ..()
+	if(H.mind)
```

Comment:
```suggestion
/datum/outfit/admin/post_equip(mob/living/carbon/human/H, visualsOnly)
	. = ..()
	if(visualsOnly)
		return

	if(H.mind)
```

Reply:
This can be comitted into your branch with no changes tourte, visualsOnly is never actually set to true which is hillarious as well
Might want to do visualsOnly = FALSE in the arg tho